databases/week1/data.sql

```diff
@@ -0,0 +1,47 @@
+-- 1 task
+SELECT count(id)
+FROM task
+-- 2 task
+SELECT title, due_date
+FROM task
+WHERE due_date IS NULL;
+-- 3 task
+SELECT *
+FROM TASK
+WHERE STATUS_ID = 3
+-- 4 task
+SELECT *
+FROM TASK
+WHERE STATUS_ID != 3
+-- 5 task
+SELECT *
+FROM TASK
+ORDER BY created ASC
+-- 6 task
+SELECT *
```

Comment:
the query should list only one task but seems like you are trying to show all the tasks in DESC

Reply:
as I use Limit 1 in the end which only gives 1, it was commented out ! sorry about that. 
I have tried with Top 1 but doesnt work, is there any other suggestion from you